src/wiktextract/wiktionary.py

```diff
@@ -225,7 +225,7 @@ def reprocess_wiktionary(
             last_time = estimate_progress(
                 processed_pages, all_page_nums, start_time, last_time
             )
-    if wxr.config.extract_thesaurus_pages:
+    if wxr.config.dump_file_lang_code == "en":
         emit_words_in_thesaurus(wxr, emitted, out_f, human_readable)
```

Comment:
`emit_words_in_thesaurus()` is only used for English Wiktionary because other extractors now use `lang_name` property but the English extractor uses `lang`.